README.md

```diff
@@ -1,6 +1,6 @@
 # Hera ![Status](https://circleci.com/gh/ewasm/hera.svg?style=shield&circle-token=:circle-token)
 
-Hera is an [ewasm] virtual machine implemented in C++ conforming to [EVMC].
+Hera is an [ewasm] (revision 4) virtual machine implemented in C++ conforming to [EVMC] ABIv5.
```

Comment:
I'd refer to it just as `EVMC 5`.

Reply:
I'm not sure, I think that is not enough. Perhaps `EVMC v5`? Still looks to me that  `EVMC ABIv5` is the most expressive one.

Reply:
But how do you know which versions it refers to?

Reply:
Should have an explanatory section in the EVMC documentation which explains the ABI version, how it relates to EVMC versioning and the preferred way to refer to a given "ABI version".

Reply:
Created an issue on the evmc repo to track this.